content/premium.md

```diff
@@ -129,7 +129,7 @@ When hosting YAGPDB yourself, you have access to a few methods to source premium
   1. Configure your price per premium slot in `premium/patreonpremiumsource/patreonpremiumsource.go > func
      CalcSlotsForPledge`.
   2. Configure relevant env variables to connect YAGPDB to your Patreon API.
-- Generating premium codes with `GeneratePremiumCode` [command](commands#generatepremiumcode).
+- Generating premium codes with `GeneratePremiumCode` [command](core/all-commands#generatepremiumcode).
```

Comment:
```suggestion
- Generating premium codes with the [`GeneratePremiumCode` command](core/all-commands#generatepremiumcode).
```
This can surely be expanded a bit — you already write `[X Command](...)` elsewhere, and that way it is easier to click on a mobile screen with sausage fingers.